engine/gamesys/src/gamesys/scripts/script_http.cpp

```diff
@@ -309,6 +309,7 @@ namespace dmGameSystem
 
             g_Service = dmHttpService::New(&params);
             dmScript::RegisterDDFDecoder(dmHttpDDF::HttpResponse::m_DDFDescriptor, &HttpResponseDecoder);
+            dmScript::RegisterDDFDecoder(dmHttpDDF::HttpRequestProgress::m_DDFDescriptor, &HttpRequestProgressDecoder);
```

Comment:
Since we have two ways of creating messages, with the arrays/strings in-place, or as offsets as payloads.
Our current implementation of DoSaveMessage save them in-place, whereas the handler in comp_script treats them as being offsets.

This is a confusing issue, and we should strive for unifying this mechanic. E.g. storing a setting in the binary blob indicating the type of storage.
Or always preferring offsets.